src/main/kotlin/net/maizegenetics/phgv2/api/HaplotypeGraph.kt

```diff
@@ -89,6 +89,10 @@ class HaplotypeGraph(hvcfFiles: List<String>) {
         return gameteSet
```

Comment:
You're changing HaplotypeGraph?

Reply:
Yes, sorry forgot I added this in a week ago.  I am adding a helper function and making the hapIdsToSampleGametes function more efficient by removing the reduce.